packages/opentelemetry-instrumentation/src/platform/node/instrumentation.ts

```diff
@@ -102,6 +102,30 @@ export abstract class InstrumentationBase<T = any>
     return exports;
   }
 
+  protected getModuleVersion(directory: string): string {
+    const modulePackage = this.findModulePackage(directory);
+    if (!modulePackage) {
+      return '0.0.0';
+    }
+    return modulePackage.version;
+  }
+
+  protected findModulePackage(directory: string): any {
+    const parentDirectory = path.dirname(directory)
+    if (directory === parentDirectory) { return; }
+    let contents = null
+    try {
+      contents = this.loadModulePackage(directory)
+    } catch (err) {
+    }
+    if (contents) { return contents }
+    return this.findModulePackage(parentDirectory)
+  }
+
+  protected loadModulePackage(directory: string): any {
```

Comment:
i would prefer that it return `unknown` to force typechecking

Reply:
Updated to unknown